kvdb/etcd_test.go

```diff
@@ -0,0 +1,212 @@
+// +build kvdb_etcd
+
+package kvdb
+
+import (
+	"testing"
+
+	"github.com/lightningnetwork/lnd/kvdb/etcd"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	bkey = etcd.BucketKey
+	bval = etcd.BucketVal
+	vkey = etcd.ValueKey
+)
+
+func TestBucketCreation(t *testing.T) {
+	t.Parallel()
+
+	f := etcd.NewEtcdTestFixture(t)
+	defer f.Cleanup()
+
+	testBucketCreation(t, f.NewBackend())
```

Comment:
These calls now all look extremely similar, especially when looking at the added `bbolt` tests in a later commit. And since all these test logic functions have the same signature, couldn't we add them to a list and then just loop through them for `etcd` and `bbolt`? Sort of like our itests that we run against different backends. Could just skip the `require.Equal(t, expected, f.Dump())` part for `bbolt`...

Reply:
Done